.github/workflows/generate.yml

```diff
@@ -27,16 +27,20 @@ jobs:
         run: |
           python do.py setup
           python do.py init
+      - name: Generate Python and Go SDK
+        run: |
+          ".env/bin/python" do.py generate_sdk
```

Comment:
the problem is see if we move the openapiart generation code to do.py that we need use env otherwise we wont be getting dependencies 
should we create a different file altogether and then call it in do.py ?